src/main/java/com/direwolf20/buildinggadgets/common/schema/template/package-info.java

```diff
@@ -0,0 +1,7 @@
+@MethodsReturnNonnullByDefault
+@ParametersAreNonnullByDefault
+package com.direwolf20.buildinggadgets.common.schema.template;
+
+import mcp.MethodsReturnNonnullByDefault;
+
+import javax.annotation.ParametersAreNonnullByDefault;
```

Comment:
new line for git

Reply:
I still don't understand what you mean... Though maybe you've answered already above...

Reply:
It's common standard to add an empty new line at the end of your files when working with git. https://stackoverflow.com/a/5813359